py-polars/polars/interchange/column.py

```diff
@@ -0,0 +1,68 @@
+import enum
+from typing import Any, Dict, Iterable, Optional, Tuple
+
+import polars as pl
+from polars.interchange.buffer import _IXBuffer
+
+
+class _IXColumn:
+    def __init__(self, column: pl.Series, allow_copy: bool = True) -> None:
+        if not isinstance(column, pl.Series):
+            raise ValueError("`column` is not a Polars Series")
+
+        self._col = column
+        self._allow_copy = allow_copy
+
+    @property
+    def size(self) -> Optional[int]:
+        # TODO: elements
+        # return len(self._col)
+        return None
+
+    @property
+    def offset(self) -> int:
```

Comment:
So assuming our `Column` contains chunks, we'd need to get the offset here instead of just returning 0 like pandas.

From the API docstring verbatim:
> Offset of first element.
> May be > 0 if using chunks; for example for a column with N chunks of
> equal size M (only the last chunk may be shorter),
> ``offset = n * M``, ``n = 0 .. N-1``.

Reply:
related: https://github.com/data-apis/dataframe-api/issues/67#issuecomment-924841597

Reply:
I'll resurface when ready.